File: ios/CropperViewManager.m
//
//  CropperViewManager.m
//  bloomTest23
//
//  Created by Nika Samadashvili on Dec/26/19.
//  Copyright © 2019 Facebook. All rights reserved.
//


#import "React/RCTViewManager.h"
@interface RCT_EXTERN_MODULE(CropperViewManager, RCTViewManager)
  RCT_EXPORT_VIEW_PROPERTY(proportion, NSString)
  RCT_EXPORT_VIEW_PROPERTY(onDone, RCTDirectEventBlock)
@end

File: ios/CropperViewManager.swift
//
//  CropperViewManager.swift
//  bloomTest23
//
//  Created by Nika Samadashvili on Dec/26/19.
//  Copyright © 2019 Facebook. All rights reserved.
//

import Foundation

@objc(CropperViewManager)
class CropperViewManager: RCTViewManager {
  override func view() -> UIView! {
    return CropperConatainerView()
  }
  
  
  override static func requiresMainQueueSetup() -> Bool {
     return true
   }
}
